components/build-service/base/add-ca-bundle-patch.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Why these files have been added to base? If it's intentional, it missed entries in kustomize

Reply:
mistake. I removed them.